### PR TITLE
edited all classes to use basemodel instead of remodel (code review?)

### PR DESCRIPTION
**Pull request by sungbae**

### CODE REVIEW

**Comment by untra** [comment]
@sungbae this is spectacular! You crushed this! Lets CR this later today! :icecream:

File: src/models/student.py [hunk 1]
```diff
@@ -22,13 +29,17 @@ def is_minor(data):
 
     # must be overridden
     def requiredFields():
-        super + []
+        super + ['student_id', 'email', 'college', 'majors', 'minors', 'gpa', 'course_history', 'credits_earned']
```
Comment (untra) [inline src/models/student.py]: remove email, majors, minors, gpa, course_history, credits_earned

File: src/models/section.py [hunk 2]
```diff
@@ -1,30 +1,20 @@
...
+            'section_id' : (is_int, ),
+            'course_id' : (is_int, ),
+            'course_name' : (is_str, ),
+            'average_grade' : (is_int, ),
+            'credit_hours' : (is_int, ),
+            'section_time' : (is_time_string, ),
+            'instructor_id' : (is_int, )
+        })
+
+    def is_time_string(data):
```
Comment (untra) [inline src/models/section.py]: remove

File: src/models/user.py [hunk 3]
```diff
@@ -5,18 +5,19 @@
...
-    USER_ETHNICITIES        = ['Caucasian', 'Black', 'Asian', 'Native American', 'Pacific Islander', 'Hispanic or Latino', 'Multiracial', 'Other', 'Prefer Not to Disclose']
-    USER_NATIVE_LANGUAGES   = ['English', 'Spanish', 'French', 'German', 'Chinese', 'Japanese', 'Russian', 'Arabic', 'Portuguese', 'Hindi']
+    USER_GENDERS            = ['Male', 'Female']
+    USER_ETHNICITIES        = ['American Indian or Alaska Native', 'Asian', 'Black or African American', 'Hispanic or Latino', 'Native Hawaiian or Other Pacific Islander', 'White', 'Other']
+    USER_NATIVE_LANGUAGES   = ['English', 'Spanish', 'French', 'German', 'Korean', 'Chinese', 'Japanese', 'Russian', 'Arabic', 'Portuguese', 'Hindi']
 
     # must be overridden
     def requiredFields():
-        super + ['registration', 'username', 'email', 'accepted_tos']
+        super + ['registration', 'user_id',  'username', 'email', 'gender', 'accepted_tos']
```
Comment (untra) [inline src/models/user.py]: @untra is working on user

File: src/models/section.py (hunk 2 above)
Comment (sungbae) [inline src/models/section.py]: Actually nvm, i'll just remove this.

**Pull request merged**